precompiles/avs/avs_test.go

```diff
@@ -210,18 +207,54 @@ func (suite *AVSManagerPrecompileSuite) TestRegisterAVS() {
 func (suite *AVSManagerPrecompileSuite) TestDeregisterAVS() {
 	avsName := "avsTest"
 	commonMalleate := func() (common.Address, []byte) {
-		// prepare the call input for delegation test
+		// prepare the call input for avs test
 		input, err := suite.precompile.Pack(
 			avs.MethodDeregisterAVS,
 			suite.Address,
 			avsName,
 		)
 		suite.Require().NoError(err, "failed to pack input")
-		return common.HexToAddress("0x3e108c058e8066DA635321Dc3018294cA82ddEdf"), input
+		return suite.Address, input
 	}
 	successRet, err := suite.precompile.Methods[avs.MethodDeregisterAVS].Outputs.Pack(true)
 	suite.Require().NoError(err)
+	setUp := func() {
+		slashAddress, rewardAddress := "0xDF907c29719154eb9872f021d21CAE6E5025d7aB", "0xDF907c29719154eb9872f021d21CAE6E5025d7aB"
+		avsOwnerAddress := []string{
+			sdk.AccAddress(suite.Address.Bytes()).String(),
+			sdk.AccAddress(utiltx.GenerateAddress().Bytes()).String(),
+			sdk.AccAddress(utiltx.GenerateAddress().Bytes()).String(),
+		}
+		assetID := suite.AssetIDs
+		minStakeAmount, taskAddr := uint64(3), "0xDF907c29719154eb9872f021d21CAE6E5025d7aB"
+		avsUnbondingPeriod, minSelfDelegation := uint64(3), uint64(3)
+		epochIdentifier := epochstypes.DayEpochID
+		params := []uint64{2, 3, 4, 4}
+		avs := &types.AVSInfo{
+			Name:                avsName,
+			AvsAddress:          suite.Address.String(),
+			SlashAddr:           slashAddress,
+			RewardAddr:          rewardAddress,
+			AvsOwnerAddress:     avsOwnerAddress,
+			AssetIDs:            assetID,
+			AvsUnbondingPeriod:  avsUnbondingPeriod,
+			MinSelfDelegation:   minSelfDelegation,
+			EpochIdentifier:     epochIdentifier,
+			StartingEpoch:       1,
+			TaskAddr:            taskAddr,
+			MinStakeAmount:      minStakeAmount,
+			MinOptInOperators:   params[0],
+			MinTotalStakeAmount: params[1],
+			AvsReward:           sdk.MustNewDecFromStr(strconv.FormatUint(params[1], 10)),
```

Comment:
consider using **strconv.Itoa(int(xxx))** instead of calling _strconv.FormatUint(xxx, 10)_, as it is more concise and performs better.